packages/@react-aria/table/test/tableResizingTests.tsx

```diff
@@ -527,6 +527,23 @@ export let resizingTests = (render, rerender, Table, ControlledTable, resizeCol,
         expect(getColumnWidths(tree)).toStrictEqual([113, 112, 113, 112, 450]);
       });
 
+      it('onResize end called with values even if no resizing took place uncontrolled', function () {
+        let allowsResizing = true;
+        let columns = [
+          {name: 'Name', id: 'name', allowsResizing},
+          {name: 'Type', id: 'type', defaultWidth: '3fr', allowsResizing},
+          {name: 'Level', id: 'level', allowsResizing},
+          {name: 'Height', id: 'height', defaultWidth: '5fr', allowsResizing}
+        ];
+        let columnNames = ['Name', 'Type', 'Level', 'Height'];
+        let onResizeEnd = jest.fn();
+        let tree = render(<Table columns={columns} rows={rows} onResizeEnd={onResizeEnd} />);
+        expect(getColumnWidths(tree)).toStrictEqual([90, 270, 90, 450]);
+        resizeCol(tree, 'Type', 0);
+        expect(getColumnWidths(tree)).toStrictEqual([90, 270, 90, 450]);
+        expect(onResizeEnd).toHaveBeenCalledWith(mapFromWidths(columnNames, [90, 270, '1fr', '5fr']));
```

Comment:
@snowystinger should we have a test where there is initially no columns and then they get added like in RAC? I saw there were already tests for adding columns which is why I was confused why the tests didn't catch the issue. Maybe because we didn't try to do some interactions on them after adding?

Reply:
I think you are right, i'll edit one of them to account for it Monday if it's not added before then

Reply:
I added more tests and found more issues, see https://github.com/adobe/react-spectrum/pull/5355 which I split off from this in case we want to address those issues separately.